### PR TITLE
Only load the XSL once.

### PR DESCRIPTION
**Pull request by richardschneider**
A compiled XSL is thread safe.  Just load and compile in the static initialiser.

This change improved `docfx metadata` by 2 seconds.  Before it took ~12s now its down to 10s.

### CODE REVIEW

**Comment by vicancy** [comment]
How about using Singleton?

**Comment by richardschneider** [comment]
Yes, it could be just a `static` class.  Just didn't want to change the caller.  Will do later (in an hour or so)

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentTransformer.cs [hunk 1]
```diff
@@ -12,14 +12,15 @@ namespace Microsoft.DocAsCode.Metadata.ManagedReference
...
     public class TripleSlashCommentTransformer
     {
-        private readonly XslCompiledTransform _transform;
+        private static readonly XslCompiledTransform _transform;
 
-        public TripleSlashCommentTransformer()
+        static TripleSlashCommentTransformer()
         {
-            var assembly = this.GetType().Assembly;
+            var assembly = Assembly.GetExecutingAssembly();
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentTransformer.cs]: `typeof(TripleSlashCommentTransformer).Assembly`?

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentTransformer.cs [hunk 2]
```diff
@@ -12,14 +12,15 @@ namespace Microsoft.DocAsCode.Metadata.ManagedReference
 
     using Microsoft.DocAsCode.Common;
     using Microsoft.DocAsCode.DataContracts.ManagedReference;
+    using System.Reflection;
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentTransformer.cs]: move namespace to 'System' group?

**Pull request merged**